src/subapps/dataExplorer/DataExplorer.tsx

```diff
@@ -4,41 +4,23 @@ import { notification } from 'antd';
 import { isString } from 'lodash';
 import React, { useReducer } from 'react';
 import { useQuery } from 'react-query';
-import { getResourceLabel } from '../../shared/utils';
 import { DataExplorerTable } from './DataExplorerTable';
 import './styles.less';
 import { ProjectSelector } from './ProjectSelector';
-import {
-  CONTAINS,
-  EMPTY_VALUE,
-  PredicateFilterT,
-  PredicateSelector,
-  doesResourceContain,
-  isPathMissing,
-} from './PredicateSelector';
-import { normalizeString } from 'utils/stringUtils';
+import { PredicateSelector } from './PredicateSelector';
 
 export interface DataExplorerConfiguration {
   pageSize: number;
   offset: number;
   orgAndProject?: [string, string];
-  predicatePath: string | null;
-  predicateFilter: PredicateFilterT | null;
-  predicateValue: string | null;
+  predicateFilter: ((resource: Resource) => boolean) | null;
```

Comment:
I think it make sense if the `predicateFilter` can be rename it to `predicate`

Reply:
this too.